package-lock.json

```diff
@@ -5054,7 +5054,8 @@
       "dependencies": {
         "@formatjs/intl-relativetimeformat": {
           "version": "4.1.1",
-          "bundled": true,
+          "resolved": "https://registry.npmjs.org/@formatjs/intl-relativetimeformat/-/intl-relativetimeformat-4.1.1.tgz",
```

Comment:
this shouldn't be included.

Reply:
ok, removed :)

Reply:
This is caused by devs using different versions of npm, either way is fine for future reference. That said, we should probably all update to an agreed minimum version to avoid this unnecessary noise in future changes.